src/sb_vision/src/HSVFilter.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Takes in an image and transforms it into a binary image
+ * given some color specification.
+ * Author: Valerian Ratu
+ * Ref: 	Color Picker
+ *			https://raw.githubusercontent.com/kylehounslow/opencv-tuts/master/auto-colour-filter/AutoColourFilter.cpp
+ *		Color Bar:
+ *			http://opencv-srf.blogspot.ca/2010/09/object-detection-using-color-seperation.html
+ */
+
+#include "../include/HSVFilter.h"
```

Comment:
not sure if the convention is `include_library( )` or `include_directories( )` to include headers in the `CMakeLists.txt` but I think will be better than using relative path for headers

Reply:
Good catch! We should definitly be using relative includes here (which works because of the exact reason you mentioned). :+1: Changed